core/templates/dev/head/components/forms/schema_editors/SchemaBasedChoicesEditorDirective.js

```diff
@@ -37,7 +37,11 @@ oppia.directive('schemaBasedChoicesEditor', [
         'schema_based_choices_editor_directive.html'),
       restrict: 'E',
       compile: NestedDirectivesRecursionTimeoutPreventionService.compile,
-      controller: ['$scope', function($scope) {
+      controller: ['$scope', '$window', function($scope, $window) {
+        if ($scope.schema().value && $scope.schema().value === 'srl' &&
+            !$window.webkitSpeechRecognition) {
+          $scope.isDisabled = true;
```

Comment:
isDisabled should be a function as it's called using `isDisabled()` from the template

Reply:
Ah! my bad. I have fixed that.